app/helpers/application_helper.rb

```diff
@@ -9,4 +11,36 @@ def format_full_date(date)
   def format_month_year_date(date)
     (date || NullDate.new).strftime(MONTH_YEAR_DATE_FORMAT_STRING)
   end
+
+  def en_ar_country(country)
+    en_country = ISO3166::Country[country]
+    ar_country = I18n.with_locale :ar do
+      t country, default: :missing
+    end
+
+    ("(#{en_country}) #{ar_country}").html_safe
+  end
+
+  def country_select_list
```

Comment:
Yes. There is a gem for everything! :( 
It doesn't feel like real programming. It feels more like just playing with lego.